Classes/SyslogBackend.php

```diff
@@ -75,6 +76,29 @@ public function append($message, $severity = LOG_INFO, $additionalData = NULL, $
 		syslog($severity, $output);
 	}
 
+	/**
+	 * Fetches the client IP address, also taking care of upstream proxies, if needed
+	 *
+	 * @return string|null
+	 */
+	protected function getClientIPAddress() {
+		if (isset($_SERVER['REMOTE_ADDR'])) {
+			$remoteAddr = new \IpUtils\Address\IPv4($_SERVER['REMOTE_ADDR']);
```

Comment:
This external package is currently required in daz-base (`"xrstf/ip-utils": "1.1.*"`). Since SysLog is a standalone package, we should require it here too.